stdlib/contrib/RohanSreerama5/naiveBayesClassifier/script.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
Can you make this script part of the README for the package? That way its easier for people to discover?